JniBridge.bee.cs

```diff
@@ -388,10 +443,168 @@ static void SetupTestProgramOsx(ToolChain toolchain, NativeProgram staticLib, Co
         np.Libraries.Add(new DynamicLibrary(javaLibDir.Combine("libjvm.dylib")));
         np.CompilerSettings().Add(c => c.WithCustomFlags(new [] {$"-Wl,-rpath,{javaLibDir}"}));
         
-        var destDir = new NPath("build").Combine("osx");
+        var destDir = new NPath("build").Combine(Platform.OSX);
         var config = new NativeProgramConfiguration(codegen, toolchain, false);
         var target = np.SetupSpecificConfiguration(config, config.ToolChain.ExecutableFormat).DeployTo(destDir);
         
-        Backend.Current.AddAliasDependency("build:osx:test", target.Paths);
+        Backend.Current.AddAliasDependency($"build:{Platform.OSX}:test", target.Paths);
+    }
+
+    static NativeProgram SetupTestProgramWindows(ToolChain toolchain, NativeProgram staticLib, CodeGen codegen, NPath generatedFilesDir, Jdk jdk, out NPath targetExecutable)
+    {
+        var np = new NativeProgram("JNIBridgeTests");
+        np.Sources.Add(new NPath("test").Files("*.cpp"));
+        np.IncludeDirectories.Add(generatedFilesDir);
+        np.IncludeDirectories.Add(jdk.JavaHome.Combine("include"));
+        np.IncludeDirectories.Add(jdk.JavaHome.Combine("include", "win32"));
+        np.Libraries.Add(staticLib);
+        var javaLibDir = jdk.JavaHome.Combine("lib");
+        np.Libraries.Add(new StaticLibrary(javaLibDir.Combine("jvm.lib")));
+
+        var destDir = new NPath("build").Combine(Platform.Windows);
+        var config = new NativeProgramConfiguration(codegen, toolchain, false);
+        var target = np.SetupSpecificConfiguration(config, config.ToolChain.ExecutableFormat).DeployTo(destDir);
+
+        var exeName = np.Name + ".exe";
+        var pdbName = np.Name + ".pdb";
+        var server = jdk.JavaHome.Combine("jre/bin/server");
+        targetExecutable = Backend.Current.SetupCopyFile(server.Combine(exeName), destDir.Combine(exeName));
+        var targetPdb = Backend.Current.SetupCopyFile(server.Combine(pdbName), destDir.Combine(pdbName));
+        var targetJar = Backend.Current.SetupCopyFile(server.Combine("jnibridge.jar"), "build/jnibridge.jar");
```

Comment:
Just noticed this: is this copying jnibridge.jar to JDK or the opposite why (meaning a non-existent file gets copyied)?